x-pack/plugins/cases/server/client/configure/client.ts

```diff
@@ -436,8 +436,13 @@ export async function create(
       fieldName: 'customFields',
     });
 
-    await validateTemplates({
+    const updatedTemplates = transformTemplateCustomFields({
```

Comment:
I was thinking that, only for the create scenario, it may be fine if we throw a bad request error because the user tries to create a configuration with invalid custom fields in templates. Wdyt?

Reply:
I thought about it, and there are 2 aspects where  it might be confusing behaviour:
- if other fields in template are not there like (case title or description) we will throw validation error, but not for custom fields
- when user will create a case without template and try to add custom field, it will throw validation error